src/main/java/sirius/biz/tenants/TenantExportJobFactory.java

```diff
@@ -37,4 +58,18 @@ public String getCategory() {
     protected boolean hasPresetFor(QueryString queryString, Object targetObject) {
         return queryString.path().startsWith("/tenant");
     }
+
+    @Override
+    protected void collectParameters(Consumer<Parameter<?>> parameterCollector) {
+        super.collectParameters(parameterCollector);
+        parameterCollector.accept(permissionsParameter);
+    }
+
+    @Override
+    protected boolean includeEntityDuringExport(E entity, ProcessContext processContext) {
+        return processContext.getParameter(permissionsParameter)
+                             .map(necessaryPermissions -> entity.getPermissions()
+                                                                .containsAll(List.of(necessaryPermissions.split(","))))
+                             .orElse(true);
+    }
```

Comment:
**Question:** Is it possible to split the tenant export factory and job and move this to the job so we can remember the list of split permissions in the context of the job execution? Seems a bit excessive to re-split it for each tenant.

Reply:
The split is now performed in the parameter, and the parameter value is used directly.